src/components/Executions/contexts.ts

```diff
@@ -16,3 +18,7 @@ export const NodeExecutionsContext = React.createContext<
 export const NodeExecutionsRequestConfigContext = React.createContext<
     RequestConfig
 >({});
+
+export const ExecutionDataCacheContext = React.createContext<
+    ExecutionDataCache
+>({} as ExecutionDataCache);
```

Comment:
Same comment here as above

Reply:
Resolving with the same answer as above.